flink-tests/src/test/java/org/apache/flink/test/recovery/JobManagerHAProcessFailureBatchRecoveryITCase.java

```diff
@@ -212,7 +216,8 @@ public void flatMap(Long value, Collector<Long> out) throws Exception {
 	}
 
 	@Test
-	public void testJobManagerProcessFailure() throws Exception {
+	public void testDispatcherProcessFailure() throws Exception {
```

Comment:
This case fails on travis test of FLINK-10397 #6752 , thus highly unstable.

Reply:
Thanks for pointing this out @TisonKun. I will look into it.

Reply:
I think I've found the culprit. It seems as if the `RestClient` does not time out an established HTTP connection to the server. Due to that, it can happen that we just established the connection when the `DispatcherProcess` is killed. This apparently won't be noticed by Netty in time for the test to finish. I'm still investigating why Netty cannot detect an unreachable remote host.

Reply:
FYI: https://issues.apache.org/jira/browse/FLINK-10415